package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "hoelbettin-frontend",
+  "author": "Miro Nieminen <miro.nieminen@futurice.com>",
+  "license": "MIT",
+  "dependencies": {
+    "babel-plugin-styled-components": "^1.1.7",
+    "es6-promise": "^4.1.1",
+    "immutable": "^3.8.2",
+    "isomorphic-fetch": "^2.2.1",
+    "lodash": "^4.17.4",
+    "loglevel": "^1.5.1",
+    "next": "latest",
```

Comment:
This seems like a bad idea, since next has been releasing breaking major versions quite often

Reply:
Yeah, also next 4 claims to only support react 16, but we're depending on 15. Should we downgrade to next 3.2.2 or would that break other stuff?